### ISSUE TITLE
SC 3.3.7: replace "must" by "is"

### ISSUE THREAD

**Issue by cstrobbe** [w3c/wcag#1280]
SC 3.3.7 in the [current draft (11 August 2020)](https://www.w3.org/TR/2020/WD-WCAG22-20200811/#accessible-authentication) reads (emphasis added),

> If an authentication process relies on a cognitive function test, at least one other method **must** also be available that does not rely on a cognitive function test. 

WCAG 2.x has always avoided using keywords such as "must", "must not" and "should" in the text of success criteria (unlike notes to SC and the Conformance section). Perhaps this should be reworded to the following:

> If an authentication process relies on a cognitive function test, at least one other method is also available that does not rely on a cognitive function test.

**Comment by awkawk** [comment]
Good point, @cstrobbe, I agree with the suggestion.
